static/js/settings.js

```diff
@@ -0,0 +1,115 @@
+const API_PREFIX = 'api'
+
+const baseUrl = window.location.origin
+const fields = ["session_id", "folder_path", "loop_delay_seconds", "loop_variation_percentage", "extra_ids"]
```

Comment:
Use Map https://developer.mozilla.org/it/docs/Web/JavaScript/Reference/Global_Objects/Map

Reply:
eg: jsonFieldsToId